components/ws-manager-bridge/ee/src/prebuild-updater-db.ts

```diff
@@ -79,6 +79,10 @@ export class PrebuildUpdaterDB implements PrebuildUpdater {
                 span.setTag("updatePrebuildWorkspace.prebuild.state", updatedPrebuild.state);
                 span.setTag("updatePrebuildWorkspace.prebuild.error", updatedPrebuild.error);
 
+                if (updatedPrebuild.state && updatedPrebuild.state !== "building") {
```

Comment:
I'd recommend flipping the logic - list all the "terminal" states for which we do increment the metric. It's more resilient to new states being introduced in the future. 

A better alternative yet, add an instance method on the `prebuild` object to check if it's in terminal state. That way, you can co-locate the logic around the definition of state - higher chance we'll not forget to update something when we make changes.

Reply:
> list all the "terminal" states for which we do increment the metric

Agree it would be more resilient 👍🏼